packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html

```diff
@@ -898,6 +898,14 @@
                         </tr>
                     </tfoot>
                 </table>
+                <div>
+                    <div
+                        data-translate="pack_move_offers_debit_mention_1"
+                    ></div>
+                    <div
+                        data-translate="pack_move_offers_debit_mention_2"
+                    ></div>
```

Comment:
```suggestion
                    <p
                        data-translate="pack_move_offers_debit_mention_1"
                    ></p>
                    <p
                        data-translate="pack_move_offers_debit_mention_2"
                    ></p>
```

Reply:
Sorry but if I use `<p>` tag the displayed result it's not the result I want to display.

Reply:
@Steffy29 what is the issue?

Reply:
the space between the 2 lines has a different size between `<p>` tag and `<div>` tag. I can try another solution if you want.

Reply:
Hello @Steffy29, can you try `<p class="mb-0"></p>`?